pass-an-image.cpp

```diff
@@ -2,11 +2,14 @@
 #include "brightener.h"
 
 int main() {
-    Image image;
-    image.rows = 512;
-    image.columns = 512;
-	std::cout << "Brightening a 512 x 512 image\n";
-    ImageBrightener brightener(image);
-    int attenuatedCount = brightener.BrightenWholeImage();
-    std::cout << "Attenuated " << attenuatedCount << " pixels\n";
+    try {
+        Image image(512, 512); // Create image with 512x512 size
+        std::cout << "Brightening a 512 x 512 image\n";
+        ImageBrightener brightener(image);
+        int attenuatedCount = brightener.BrightenWholeImage();
+        std::cout << "Attenuated " << attenuatedCount << " pixels\n";
+    }
+    catch (const std::exception& e) {
```

Comment:
Always catch specific exceptions. Here, you are catching a generic exception, printing it and continuing. This can hide errors, especially at debugging / development stage.

Reply:
added more specific exceptions.